### PR TITLE
Initial client

### PR DESCRIPTION
**Pull request by niels-nijens**
Added initial API client and documentation.

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+     * @return array
+     */
+    private function getRequestHeaders($url, array $parameters)
+    {
+        $headers = array();
+
+        if (isset($this->clientId) && isset($this->sharedSecret)) {
+            $objectIdentifier = null;
+            if ($this->apiVersion === '1.1' && isset($parameters['id'])) {
+                $objectIdentifier = $parameters['id'];
```
Comment (RonRademaker) [inline src/Client.php]: This implies a BC break, shouldn't the new API version be 2.0?

**Comment by RonRademaker** [comment]
Very good to see this library / PR :smile: 

Do you also plan to add (configurable) logging to this client to make it easier to trace issues?

**Comment by niels-nijens** [comment]
That's out of scope for the first version. If you'd like you can always create an issue for it with what you'd like to have logged and why.

File: src/Client.php (hunk 1 above)
Comment (niels-nijens) [inline src/Client.php]: This library is merely implementing the API. :wink:

Besides that version 1.1 is not meant as a replacement, just an alternative. That's why it wasn't versioned as 2.0. Both API versions are still available.

**Pull request merged**